README.md

```diff
@@ -40,4 +42,49 @@ api_instance
   .catch((error) => console.log(error.response.data));
 ```
 
-Returns the information about the api.
\ No newline at end of file
+Returns the information about the API.
+
+### Authentication
+
+The username and password or the authentication token can be manually specified when initializing the configuration:
+
+```javascript
+const config = new ionoscloud.Configuration({
+    username: 'YOUR_USERNAME',
+    password: 'YOUR_PASSWORD',
+    apiKey: 'YOUR_API_KEY'
+});
+```
+
+Environment variables can also be used; the SDK uses the following variables:
+
+* IONOS\_USERNAME - to specify the username used to log in
+* IONOS\_PASSWORD - to specify the password
+* IONOS\_TOKEN - if an authentication token is being used
```

Comment:
**NOTE**

The authentication is not working when using `IONOS_TOKEN`, but I'll leave this `README` as it is since we will fix this really soon in the future (it makes no sense to modify the `README` in order to bring it to the same stage later on).